docs/create-user.md

```diff
@@ -0,0 +1,52 @@
+
+# Creating a Hawtio user for Form authentication
+
+If you install Hawtio on Kubernetes or choose to use Form authentication, you need to create a Hawtio user manually.
+
+## Creating a ServiceAccount
+
+First, create a `ServiceAccount` named `hawtio-user` in the namespace you installed Hawtio.
+
+```sh
+$ cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: hawtio-user
+  namespace: hawtio
+EOF
+```
+
+## Creating a RoleBinding/ClusterRoleBinding
+
+Next, assign the user with a role by creating a `RoleBinding` or `ClusterRoleBinding` respectively depending on the [deployment mode](../README.md#deployment) you chose. For more information on RBAC, see [RBAC](./rbac.md).
+
+```sh
+$ cat <<EOF | kubectl apply -f -
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: hawtio-user
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Maybe it can be stressed that this is an example, with broad permissions that should be refined carefully.

Reply:
Thanks!  Updated.